src/action.js

```diff
@@ -3,14 +3,21 @@
 let ParamsFilter = require('./param-filter')
 let Response = require('./response')
 let ModelLoader = require('./model-loader')
+let operationConstants = ['CREATE', 'UPDATE', 'DELETE', 'VIEW', 'LIST']
+let mustHaveAttributes = ['event', 'context', 'schema', 'operation', 'body']
 
 let Action = function (config) {
   this.response = false
-  this.context = config.context
-  this.event = config.event
-  this.operation = config.operation
-  this.body = config.body
-  this.schema = config.schema
+  for (var key in config) {
+    this[key] = config[key]
+    let oneMustHave = mustHaveAttributes.indexOf(key)
+    if (-~oneMustHave) {
+      mustHaveAttributes.splice(oneMustHave, 1)
+    }
+  }
+  if (mustHaveAttributes.length > 0 || ~-operationConstants.indexOf(this.operation)) {
```

Comment:
Yes, that's a good question..
As I told you, I think we need some sort of configuration when the user is importing the module.

Something like:
```
let lamb = require('lamb-wrap')
lamb.customConfigs({
  operationsConstants: ['CUSTOM_OPERATION']
})
```

I've gotta figure it out first what will be the custom points of the module, but I guess this'll handle it.

After we decide this I can implement something as operationsConstants.concat(customConfigs.operationsConstants)

What's your opinion about that?